wgsl/index.bs

```diff
@@ -2082,45 +2082,45 @@ shift_expression
   | shift_expression SHIFT_RIGHT additive_expression
         OpShiftRightLogical or OpShiftRightArithmetic
 
-relational_expression
+and_expression
   : shift_expression
-  | relational_expression LESS_THAN shift_expression
+  | and_expression AND shift_expression
+
+exclusive_or_expression
+  : and_expression
+  | exclusive_or_expression XOR and_expression
+
+inclusive_or_expression
+  : exclusive_or_expression
+  | inclusive_or_expression OR exclusive_or_expression
+
+relational_expression
+  : inclusive_or_expression
+  | relational_expression LESS_THAN inclusive_or_expression
         OpULessThan
         OpFOrdLessThan
-  | relational_expression GREATER_THAN shift_expression
+  | relational_expression GREATER_THAN inclusive_or_expression
         OpUGreaterThan
         OpFOrdGreaterThan
-  | relational_expression LESS_THAN_EQUAL shift_expression
+  | relational_expression LESS_THAN_EQUAL inclusive_or_expression
         OpULessThanEqual
         OpFOrdLessThanEqual
-  | relational_expression GREATER_THAN_EQUAL shift_expression
+  | relational_expression GREATER_THAN_EQUAL inclusive_or_expression
         OpUGreaterThanEqual
         OpFOrdGreaterThanEqual
 
 equality_expression
   : relational_expression
-  | relational_expression EQUAL_EQUAL relational_expression
+  | equality_expression EQUAL_EQUAL relational_expression
```

Comment:
This seems like a separate change and I don't think we want to do it. You aren't allowed to do `a == b == c` in WGSL which this would allow.

Reply:
@dj2 Does it look good now?